spec/support/factory_girl.rb

```diff
@@ -7,6 +7,13 @@
     password_confirmation 'password'
   end
 
+  factory :admin, class: User do
+    email Faker::Internet.email
+    password 'password'
+    password_confirmation 'password'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.